src/main/java/core/basesyntax/Demo.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Demo {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[new Random().nextInt(5)];
```

Comment:
declare 5 as a constant, now it's a magic number (not clear why 5 and not 6, etc)